EIPS/eip-4762.md

```diff
@@ -0,0 +1,307 @@
+---
+eip: 4762
+title: Statelessness gas cost changes
+description: This EIP introduces changes in the gas schedule to reflect the costs of creating a witness. It requires clients to update their database layout to match this, so as to avoid potential DoS attacks.
+author: Guillaume Ballet (@gballet), Vitalik Buterin (@vbuterin), Dankrad Feist (@dankrad)
+discussions-to: https://github.com/ethereum/EIPs/issues/4763
```

Comment:
Please create an Ethereum Magicians forum thread for the discussions-to.  We no longer support using GitHub issues for this purpose.